src/components/dataTile/index.tsx

```diff
@@ -0,0 +1,26 @@
+import './DataTile.scss';
+
+type Props = {
+  children: string;
+  title?: string;
+  theme?: 'blue' | 'green' | 'teal' | 'yellow';
+};
+
+const DataTile = (props: Props) => {
+  const {children, title, theme} = props;
+  const tileTheme = theme ?? 'blue';
+  const defaultTitle = {
+    blue: 'Aktualna temperatura',
+    green: 'Aktualne ph wody',
+    teal: 'Status światła',
+    yellow: 'Status grzałki',
+  };
+  return (
+    <div className={`tile tile-${tileTheme.toLowerCase()}`}>
+      <p>{title ?? defaultTitle[tileTheme]}</p>
```

Comment:
zróbmy tak, że props title będzie opcjonalny, ale jak się go nie przekaże to po prostu nie będzie żadnego tytułu
wówczas wszystkie defaultTitle znikają (zresztą docelowo nie chcemy uzależniać w żaden sposób tytułu od koloru, niekoniecznie będzie 1:1 jak na figmie, DataTile tam są wyłącznie przykładowe)

Reply:
👍